source/_integrations/stookalert.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: "Stookalert"
+description: "Instructions on how to use Stookalert data within Home Assistant"
+logo: stookalert.png
+ha_category:
+  - binary-sensor
+  - environment
+ha_release: 0.103
+ha_iot_class: Cloud Polling
+---
+
+The `stookalert` sensor platform queries the [RIVM stookalert](https://www.rivm.nl/stookalert) API for unfavorable weather conditions or poor air quality. With a Stookalert, the RIVM calls on people not to burn wood. This can prevent health problems in people in the area.
+
+</div>
```

Comment:
Remove this.

Reply:
Please explain why this should be removed. In almost all integrations a line like this exists. For instance: https://www.home-assistant.io/integrations/weather.darksky/

Reply:
He means that you must remove the `div` tag